indico/modules/events/papers/templates/management/cfp_actions/_base.html

```diff
@@ -0,0 +1,23 @@
+{% extends '_module_actions_base.html' %}
+
+{% block icon_classes %}icon-file{% endblock %}
+
+{% block buttons %}
+    {% set title -%}
+        {%- if cfp.start_dt -%}
+            {%- trans %}Reschedule call for papers{% endtrans -%}
+        {%- else -%}
+            {%- trans %}Schedule call for papers{% endtrans -%}
+        {%- endif -%}
+    {%- endset -%}
+    <div class="group">
+        <a href="#" class="i-button icon-time"
+           data-href="{{ url_for('.schedule_papers_call', event) }}"
+           data-title="{{ title }}"
+           data-update="#paper-page"
+           data-ajax-dialog
+           data-replace-update>
+            {{- _('Reschedule') if cfp.start_dt else _('Schedule') -}}
```

Comment:
I think it's better to use Jinja `trans` block if possible. It would look slightly more verbose, though.

```jinja
{% if cfp.start_dt %}
    {% trans %}Reschedule{% endtrans %}
{% else %}
    {% trans %}Schedule{% endtrans %}
{% endif %}
```